.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-
+.vscode/settings.json
```

Comment:
Hey Orlando, what does this do?

Reply:
I don't want to push the `.vscode` folder. I don't think it is something we should have remotely. I moved it here because I enabled JSX emmet autocompletion. :0